hopr/hopr-lib/src/config.rs

```diff
@@ -1,7 +1,7 @@
 use hopr_primitive_types::prelude::*;
 pub use hopr_transport::config::{
-    HoprPacketPipelineConfig, HoprProtocolConfig, HostConfig, HostType, NetworkConfig, ProbeConfig,
-    SessionGlobalConfig, TransportConfig, looks_like_domain,
+    HoprPacketPipelineConfig, HoprProtocolConfig, HostConfig, HostType, ProbeConfig, SessionGlobalConfig,
```

Comment:
`ProbeConfig` can be reduced by removing the no-longer applicable configuration options.